packages/edgeidentity/js/Identity.js

```diff
@@ -74,4 +85,14 @@ module.exports = {
    removeIdentity(item: IdentityItem, namespace: string) {
     RCTAEPEdgeIdentity.removeIdentity(item, namespace);
   },
-};
\ No newline at end of file
+};
+
+  function toIdentityMap(idObj: Object) {
+  var idMap = new IdentityMap();
+
+  for (const [key, value] of Object.entries(idObj)) {
+    idMap.items[key] = value;   
```

Comment:
It looks like value should also be converted to IdentityItem and then you can use idMap.addItem(item, key) here. 
Also, if possible we should make items read-only in the IdentityMap.